src/publisher/index.ts

```diff
@@ -76,7 +83,12 @@ export class Publisher {
   settings!: PublishOptions;
   queue: Queue;
   orderedQueues: Map<string, OrderedQueue>;
+  flowControl: FlowControl;
+
   constructor(topic: Topic, options?: PublishOptions) {
+    this.flowControl = new FlowControl(
+      options?.flowControlOptions ?? flowControlDefaults
```

Comment:
I think `||` would be safer than `??` here.  If someone passed `''` or `0` (I have no idea why they would), the only difference is `||` would correctly use the flow defaults.